misc/config_tools/scenario_config/vm_configurations_c.py

```diff
@@ -9,9 +9,9 @@
 import scenario_cfg_lib
 
 C_HEADER = scenario_cfg_lib.HEADER_LICENSE + r"""
-#include <x86/vm_config.h>
+#include <asm/vm_config.h>
 #include <vuart.h>
-#include <x86/pci_dev.h>
+#include <asm/pci_dev.h>
```

Comment:
Do you regenerate the files under  misc\config_tools\data\generic_board\generic_code\<scenario_name> to reflect your changes of config_tools?